backfiller/backfiller/main.py

```diff
@@ -231,7 +231,7 @@ def __init__(self, base_dir, channels, qualities, static_nodes=[],
 	def stop(self):
 		"""Shut down all workers and stop backfilling."""
 		self.logger.info('Stopping')
-		for node in self.workers.keys():
+		for node in list(self.workers.keys()):
```

Comment:
Do we really need a list here?

Reply:
Yes, becasuse `self.workers` is concurrently modified while we're looping, so we need to make a copy of the list as it was at the start of the loop.